feedback/static/html/studio_view.html

```diff
@@ -23,6 +23,22 @@
             <span class="tip setting-help">{% trans "The display name for this component." %}</span>
         </li>
 
+        <li class="field comp-setting-entry is-set">
```

Comment:
What is `comp` meaning?

Reply:
`comp-setting-entry` is selector that used to style all form fields.

Reply:
Great to understand this, but I asked about `comp` meaning. What is the long form of the word? It's company or comprehension, or compatibility?

Reply:
I understood you) It means `component` or `components` depending on context. 
I looked and found it (`comp-setting-entry`) is used in many studio templates.